SWLOR.Game.Server/Service/BaseService.cs

```diff
@@ -1380,7 +1380,7 @@ public static int CalculateResourceCapacity(Guid pcBaseID)
                                       var baseStructure = DataService.BaseStructure.GetByID(x.BaseStructureID);
 
                                       return baseStructure.Storage + x.StructureBonus;
-                                  }) * 0.01f;
+                                  }) * 0.03f;
```

Comment:
Need max numbers on this too please.

Reply:
This is for resource silo, I think the highest I've gotten on a silo is around 13. So it would be a 39% increase instead of what is currently a 13% increase.

Reply:
I think I'm ok with this one.